src/Components/GEOSldas_GridComp/GEOSmetforce_GridComp/LDAS_Forcing.F90

```diff
@@ -5442,34 +5362,64 @@ subroutine GEOS_openfile(FileOpenedHash, fname_full, fid, tile_coord, m_hinterp,
     
   end subroutine GEOS_openfile
 
-  subroutine get_bilinear_neighbors(m_hinterp, tile_coord, i1, j1, ll_lon, ll_lat, dlon, dlat, N_lon, N_lat, i2, j2, x1, y1, x2, y2, rc) 
+  subroutine get_neighbor_index(m_hinterp, tile_coord, ll_lon, ll_lat, dlon, dlat, N_lon, N_lat, i1, j1, i2, j2, x1, y1, x2, y2, isERA5, rc) 
     ! compute indices of nearest neighbors needed for bilinear
     ! interpolation from GEOSgcm grid to tile space
     integer, intent(in) :: m_hinterp
     type(tile_coord_type), dimension(:), intent(in)    :: tile_coord
-    integer,dimension(:), intent(in)  :: i1, j1
     real, intent(in) :: ll_lat, ll_lon, dlat, dlon
     integer, intent(in) :: N_lon, N_lat
-    integer,dimension(:), intent(out) :: i2, j2
-    real,   dimension(:), intent(out) :: x1, x2, y1, y2
+    integer, dimension(:), intent(inout) :: i1, j1
+    integer, dimension(:), intent(inout) :: i2, j2
+    real,    dimension(:), intent(out) :: x1, x2, y1, y2
+    logical, optional, intent(in) :: isERA5
     integer, optional, intent(out) :: rc
 
     real, dimension(:), allocatable :: xnew, ynew, tmp_lat, tmp_lon
     integer, dimension(:), allocatable :: inew, jnew
     character(len=*), parameter :: Iam="get_bilinear_neighbors" 
+    logical :: isERA5_
+    real  :: offset, factor
+
     ! ll_lon and ll_lat refer to lower left corner of grid cell
     ! (as opposed to the grid point in the center of the grid cell)
 
     ! pchakrab: For bilinear interpolation, for each tile, we need:
     !  x1, x2, y1, y2 (defining the co-ords of four neighbors) and
     !  i1, i2, j1, j2 (defining the indices of four neighbors)
+    isERA5_ = .false.
+    if (present(isERA5)) then
+       isERA5_ = isERA5
+    endif
+    offset  = 0.0
+    factor = 0.0
+    if (isERA5_) then
+       offset = 0.0001
```

Comment:
In the description of the issue #398, I was wondering if the "offset" is indeed specific to ERA5.  It may also needed for GEOS forcing data.  Having the "offset" should only make a noticeably difference for nearest-neighbor interpolation, which we never use for GEOS forcing.   Making things consistent, however, might result in a non-zero-diff change.  This probably requires a closer look and more discussion.